spec/rubocop/config_loader_spec.rb

```diff
@@ -246,6 +246,19 @@
           .to be_superset(expected.to_set)
       end
     end
+
+    context 'when a file inherits from an expanded path' do
+      let(:file_path) { '.rubocop.yml' }
+
+      before do
+        create_file('~/.rubocop.yml', [''])
```

Comment:
Yes, you are correct. I made the mistake of only reading the implementation of [`create_file`](https://github.com/mattjmcnaughton/rubocop/blob/9e7b262dc2bad89c9692dbb02e2f9d8220db32d3/spec/support/file_helper.rb#L6-L22) and missed [the `isolated_environment` hook](https://github.com/mattjmcnaughton/rubocop/blob/9e7b262dc2bad89c9692dbb02e2f9d8220db32d3/spec/rubocop/config_loader_spec.rb#L62) (implementation [here](https://github.com/bbatsov/rubocop/blob/cf0a3da2d237f8aed24ae4d7bab704f25868653e/spec/support/shared_contexts.rb#L6-L34)) around these examples. Misplaced concern on my part, please carry on! :smile: